src/sentry/migrations/0429_resolve_mismatched_team_slugs_for_scim.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.28 on 2023-04-26 21:58
+
+from django.db import migrations
+from django.utils.text import slugify
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def resolve_mismatched_team_slugs_for_scim(apps, schema_editor):
+    Team = apps.get_model("sentry", "Team")
+
+    for team in RangeQuerySetWrapperWithProgressBar(Team.objects.all()):
+        if team.idp_provisioned and team.slug != slugify(team.name):
+            team.slug = slugify(team.name)
```

Comment:
Is potentially changing team slugs going to break anything here?

Reply:
Should be fine. It only affects teams provisioned through IdP recently. Looking at redash, I only see 6 instances where the team would be renamed.

Was there something else you're concerned about?

Reply:
Nothing in particular. I'm just wary about renaming slugs since they tend to be used as primary keys externally. So these slugs aren't intended to match to existing value in idp or anything?

Reply:
The slugs aren't, the Idp uses the id that we return. I can change the name as much as I want and it'll still recognize the team.

Reply:
Actually, SCIM APIs don't even return a slug, they just return an ID and "displayName" which could be anything on the app side.